portlist/portlist_macos.go

```diff
@@ -159,6 +159,9 @@ func (im *macOSImpl) addProcesses() error {
 		}
 		return nil
 	}
+	defer lsofCmd.Process.Wait()
```

Comment:
Can we look at the `ProcessState.ExitCode()` of this and use that to populate `lsofFailed` instead? That appears to be the sandbox failure mode (vs. the whole process not starting).

Reply:
Done. PTAL